crates/dies-python-rt/src/env_manager/archive.rs

```diff
@@ -10,29 +10,29 @@ use tempfile::tempdir;
 
 pub fn extract_zip(archive: Bytes, target_dir: &Path, prefix: &str) -> Result<()> {
     let cursor = Cursor::new(archive.as_ref());
-    let mut archive = zip::ZipArchive::new(cursor)?;
+    let mut archive = zip::ZipArchive::new(cursor).context("Failed to create the archive")?;
```

Comment:
Most of these are super rare and it's not neccessary to add context to everything. They can all be covered by a single "Failed to extract archive". The underyling OS error will be printed anyway